app/components/Forms/LoginForm.js

```diff
@@ -48,16 +49,63 @@ function LoginForm(props) {
     closeMsg,
     loading
   } = props;
+
+  const { setAccessToken } = useContext(AuthContext);
+
   const [showPassword, setShowPassword] = useState(false);
 
   const handleClickShowPassword = () => setShowPassword(!showPassword);
   const handleMouseDownPassword = event => event.preventDefault();
 
+  const [loginSuccess, setLoginSuccess] = useState(false);
+
+  const handleLoginSubmit = async (values) => {
+    try {
+      // Send a POST request to the API endpoint
+      const response = await fetch('http://localhost:3000/auth/login', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(values),
+      });
+
+      if (response.ok) {
+        const data = await response.json();
+
+        // Extract the access token from the response
+        const accessToken = data.access_token;
+        setAccessToken(accessToken);
+        // // Store the access token in localStorage
+        // localStorage.setItem('accessToken', accessToken);
+
+        // Login successful, set the loginSuccess state to true
+
+        alert('Login Successful');
+        setLoginSuccess(true);
+      } else {
+        // Login failed, handle the error
+        alert('Incorrect Password or Email');
+        const data = await response.json();
+        console.log('Login Error:', data.error);
+        // You can display an error message to the user here if needed
+      }
+    } catch (error) {
+      console.log('Login Error:', error);
+      // Handle network or other errors here
+    }
+  };
+
+  if (loginSuccess) {
+    // Redirect to the dashboard page
+    return <Redirect to="/app/pages/table" />;
+  }
+
   return (
     <Paper className={classes.sideWrap}>
       <Hidden mdUp>
         <div className={classes.headLogo}>
-          <NavLink to="/" className={classes.brand}>
+          <NavLink to="/login" className={classes.brand}>
```

Comment:
Good practice for these paths such as `/login` is to have a utils separate file just for routing (ex.: `routes.js`) and store every possible path in there.

If you ever want to change this, you just have to change the one file, and not every occurrence of the `/login` string